lib/public/components/RunDetail/index.js

```diff
@@ -106,14 +106,10 @@ const activeFields = (model) => ({
         visible: true,
         size: 'cell-s',
     },
-    trgGlobalRunEnabled: {
-        name: 'Trigger Global Run Enabled',
+    triggerValue: {
```

Comment:
If no value is present we should display `OFF`

Reply:
Fixed in next commit